omnibus/cookbooks/omnibus-supermarket/spec/recipes/app_spec.rb

```diff
@@ -1,9 +1,8 @@
+require 'spec_helper'
+
 describe 'omnibus-supermarket::app' do
-  let(:chef_run) do
-    ChefSpec::SoloRunner.new(platform: 'ubuntu', version: '16.04') do |node|
-      node.automatic['memory']['total'] = '16000MB'
-    end.converge(described_recipe)
-  end
+  platform 'ubuntu', '16.04'
```

Comment:
I'm not familiar with `platform`, `automatic_attributes` without the context of `chef_run`; how do these get picked up?

Reply:
This is the awesomeness of ChefSpec 7.